### PR TITLE
:seedling: Unit tests for code review

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Signed-off-by: naveen <172697+naveensrinivasan@users.noreply.github.com>

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
    Unit tests for code review check.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by azeemshaikh38 — APPROVED** [review]

File: checks/code_review_test.go [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+	for _, tt := range tests {
+		tt := tt // Re-initializing variable so it is not changed while executing the closure below
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			ctrl := gomock.NewController(t)
+			mockRepo := mockrepo.NewMockRepoClient(ctrl)
+			mockRepo.EXPECT().ListMergedPRs().DoAndReturn(func() ([]clients.PullRequest, error) {
+				if tt.err != nil {
+					return nil, tt.err
+				}
```
Comment (azeemshaikh38) [inline checks/code_review_test.go:214]: Can simply `return tt.prs, tt.err` here? `if` is not needed AFAICT.

File: checks/code_review_test.go [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+				return tt.prs, tt.err
+			}).AnyTimes()
+			mockRepo.EXPECT().ListCommits().DoAndReturn(func() ([]clients.Commit, error) {
+				if tt.commiterr != nil {
+					return nil, tt.commiterr
+				}
```
Comment (azeemshaikh38) [inline checks/code_review_test.go:220]: Similar comment here.

**Review by azeemshaikh38 — APPROVED** [review]
Thanks!

File: checks/code_review_test.go (hunk 1 above)
Comment (naveensrinivasan) [reply checks/code_review_test.go]: Thanks!

File: checks/code_review_test.go (hunk 2 above)
Comment (naveensrinivasan) [reply checks/code_review_test.go]: Thanks!

**Pull request merged**